samples/5-Sites.md

```diff
@@ -0,0 +1,31 @@
+## Microsoft Graph Sites Snippets
+
+### Get root site
+```sh
+$ mgc sites item get --site-id root --select "id, displayName"
+```
+
+### Search for sites
+```sh
+$ mgc sites list --search "marketing" --select "displayName, description, webUrl"
+```
+
+### Get Id for marketing site
+```sh
+$ export sites = mgc sites list --search "marketing" --select id
```

Comment:
was this export here intentional?

Reply:
Yes. I want to add jq processing so the user can get a previously fetched site's id from the export. PowerShell has the ability to parse JSON according to the sample scripts.

Reply:
maybe add a comment to clarify the intent? and also to the next snippet (didn't realize they were related).